
From: Li Shaohua <shaohua.li@intel.com>

Possibly it doesn't change anything here (it's low mem), but it's better to
fix it.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/i386/kernel/acpi/sleep.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN arch/i386/kernel/acpi/sleep.c~acpi-flush-tlb-when-pagetable-changed arch/i386/kernel/acpi/sleep.c
--- 25/arch/i386/kernel/acpi/sleep.c~acpi-flush-tlb-when-pagetable-changed	2004-11-17 01:33:58.312451256 -0800
+++ 25-akpm/arch/i386/kernel/acpi/sleep.c	2004-11-17 01:34:48.598806568 -0800
@@ -8,7 +8,7 @@
 #include <linux/acpi.h>
 #include <linux/bootmem.h>
 #include <asm/smp.h>
-
+#include <asm/tlbflush.h>
 
 /* address in low memory of the wakeup routine. */
 unsigned long acpi_wakeup_address = 0;
@@ -27,6 +27,7 @@ static void init_low_mapping(pgd_t *pgd,
 		set_pgd(pgd, *(pgd+USER_PGDS_IN_LAST_PML4));
 		pgd_ofs++, pgd++;
 	}
+	flush_tlb_all();
 }
 
 /**
_
